### PR TITLE
CIV-10821 Use preview specific db

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

[CIV-10821
](https://tools.hmcts.net/jira/browse/CIV-10821)
### Change description ###

Use preview specific db

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by HarryH96 — DISMISSED** [review]

File: ccd-definition/CaseEvent/User/UserEvents-SDO.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0,
-    "Publish": "Y",
-    "EventEnablingCondition": "detailsOfDirection = \"Only available through tasks\""
```
Comment (mounikahmcts) [inline ccd-definition/CaseEvent/User/UserEvents-SDO.json:55]: Why do we need to edit user events in preview specific db ?

File: charts/civil-ccd/values.aat.template.yaml [hunk 2]
```diff
@@ -23,17 +25,64 @@ ccd:
...
+      secrets:
+        STORAGE_ACCOUNT_NAME:
+          secretRef: storage-secret-{{ .Release.Name }}-blobstorage
+          key: storageAccountName
+          disabled: true
+        STORAGE_ACCOUNT_KEY:
+          secretRef: storage-secret-{{ .Release.Name }}-blobstorage
+          key: accessKey
+          disabled: true
+        DEFINITION_STORE_DB_PASSWORD:
```
Comment (mounikahmcts) [inline charts/civil-ccd/values.aat.template.yaml:40]: In data store pod, why do we need to explicitly say DEFINITION_STORE_DB_PASSWORD disabled as true. I have seen ccd chart it looks only for DATA_STORE_DB_PASSWORD https://github.com/hmcts/chart-ccd/blob/master/ccd/values.yaml#L280C7-L280C29

File: charts/civil-ccd/values.aat.template.yaml [hunk 3]
```diff
@@ -23,17 +25,64 @@ ccd:
...
+          secretRef: "{{ .Values.global.postgresSecret }}"
+          key: PASSWORD
+          disabled: true
+
   ccd-definition-store-api:
     java:
       ingressHost: civil-ccd-definition-store-staging-aat.aat.platform.hmcts.net
+      environment:
+        DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.postgresPassword}}"
```
Comment (mounikahmcts) [inline charts/civil-ccd/values.aat.template.yaml:49]: Do we really data store db details in ccd-definition-store-api. It really confuses me.

**Review by jarekPierchala — DISMISSED** [review]

File: charts/civil-ccd/Chart.yaml [hunk 4]
```diff
@@ -15,7 +15,7 @@ dependencies:
     repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
     condition: civil-general-applications.enabled
   - name: ccd
-    version: 8.0.29
+    version: 9.0.20
```
[the remark this answers is not in the document]
Comment (dharmendrak) [reply charts/civil-ccd/Chart.yaml]: changed

**Review by mounikahmcts — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**